simple_rl/agents/func_approx/ddpg/DDPGAgentClass.py

```diff
@@ -126,12 +110,9 @@ def __setstate__(self, state_dictionary):
         self.critic.load_state_dict(state_dictionary["critic_state"])
         self.target_actor.load_state_dict(state_dictionary["target_actor_state"])
         self.target_critic.load_state_dict(state_dictionary["target_critic_state"])
-
         self.actor_optimizer.load_state_dict(state_dictionary["actor_optimizer"])
         self.critic_optimizer.load_state_dict(state_dictionary["critic_optimizer"])
 
-        self.replay_buffer = state_dictionary["replay_buffer"]
```

Comment:
How are we setting and getting the replay buffer?

Reply:
It's being set in the for loop in line 97 of the new code. We don't need to explicitly set it.